_posts/2020-03-18-cockpit-215.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Cockpit 215
+author: kkoukiou
+date: 2020-03-18
+tags: cockpit linux
+slug: cockpit-215
+category: release
+summary: Cockpit with Network improvements and Docker partial deprecation
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 215.
+
+### Networking: Show additional ports for each firewall zone
+
+The Firewall page now shows ports that are directly added to a zone right after the services list.
+![zone ports](/images/firewall-zone-ports.png)
+
+### Docker: Deprecate in Ubuntu and Debian developement series
```

Comment:
We also [stopped building it in Fedora ≥  32](https://github.com/cockpit-project/cockpit/pull/13702), so please generalize.